README.md

```diff
@@ -49,79 +81,94 @@ Demo is avaliable in [KVRawClientTest](https://github.com/birdstorm/KVRawClientT
 mvn clean install -Dmaven.test.skip=true
 ```
 
-### Use as maven dependency
-After building, add following lines into your `pom.xml` 
+### Add to dependency
+
+#### Use jar for binary
+
+Add your jar built with all dependencies into you project's library to use `tikv-client-java` as dependency
+
+#### Use as maven dependency
+
+After building, add following lines into your `pom.xml` if you are using Maven
+
 ```xml
 <dependency>
-    <groupId>org.tikv</groupId>
-    <artifactId>tikv-client-java</artifactId>
-    <version>2.0-SNAPSHOT</version>
+	<groupId>org.tikv</groupId>
+	<artifactId>tikv-client-java</artifactId>
+	<version>2.0-SNAPSHOT</version>
```

Comment:
Is that right? If it's not for a specific version, please make it clear.

Reply:
it is used as local dependency, which does not have a specific version yet.